includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -516,13 +536,21 @@ private function getVerticalData() {
 	}
 
 	private function getLicenseData() {
-		$licenseText = WikiFactory::getVarValueByName( 'wgRightsText', $this->wikiId ) ?: $this->wg->RightsText;
+		if ( $this->product === self::FANDOM_PRODUCT ) {
+			return [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'text',
+					'key' => 'global-footer-copyright-wikia', // TODO: add to message files - 'Copyright ' . date( 'Y' ) . ' Wikia, Inc.'
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)